index.js

```diff
@@ -1,27 +1,7 @@
 'use strict';
-const os = require('os');
 const chalk = require('chalk');
 
-const isSupported = (() => {
-	if (process.platform !== 'win32' || process.env.CI) {
-		return true;
-	}
-
-	if (process.env.ConEmuDir) {
-		return false;
-	}
-
-	// Windows 10
-	const osRelease = os.release().split('.');
-	if (
-		Number(osRelease[0]) >= 10 &&
-		Number(osRelease[2]) >= 10586
-	) {
-		return true;
-	}
-
-	return false;
-})();
+const isSupported = process.platform !== 'win32' || process.env.CI || process.env.VSCODE_PID;
```

Comment:
`process.env.VSCODE_PID` wasn't here before. Are we sure it works correctly there on Windows?

Reply:
terminal in [Visual Studio Code](https://code.visualstudio.com/)
 
![image](https://user-images.githubusercontent.com/1171691/30319134-a971feb0-97e1-11e7-9fab-279e8bd2ae7e.png)

Reply:
> Are we sure it works correctly there on Windows?
[Visual Studio Code](https://code.visualstudio.com/) Inherited emoji support from [Electron](https://electron.atom.io/)